app/models/call_log.rb

```diff
@@ -3,6 +3,8 @@ class CallLog < ApplicationRecord
 
   validates :callee_phone_number, presence: true
 
+  UNKNOWN_RESULT = 'unknown'
```

Comment:
Isn't this unnecessary?

Reply:
^^ i think the same

Reply:
I added this so that I could use the string `unknown` as a first class value

Reply:
Simplified this.